### DISCUSSION TITLE
Cant get Zigbee2mqtt to work FOH (Friends of HUE)

### DISCUSSION THREAD

**Discussion by Daniel-PT** [Koenkk/zigbee2mqtt#21289] [Devices]
Pulling my hair getting this to work. So hard to make it work :frowning:

My setup:

Unraid server - Docker Zigbee2MQTT and MQTT broker
Passhthough to docker Zigbee2MQTT
Sonoff 3.0 Zigbee controller ZBDongel plus

I was thinking to use a Lightbee FOH switch to controll my Fyrtur blinds in my licing room
BUM first road block
https://ledpaerer.dk/media/catalog/product/attachment/91-954.pdf

To get zigbee2MQTT to work with a Green power FOH you need a philips hue device to “Translate” Is there NO way to get around this?

Regards Daniel

**Comment by chris-1243** [comment]
I would like to share my experience with green device (FOH - PTM215Z from Enocean).

On a test network, I have been able to use a GreenPower device without the need of any philips Hue devices for the translation. The routers used in my test came from Ikea, Sunricher and Nodon. 
I made some automations to use the GP device to control the routers and it went fine...

Sunricher is selling some GP devices to control directly their products, Nodon as well... I only used routers with endpoint 242, input and output cluster greenPower

**Reply by Daniel-PT** [reply]
Hi @chris-1243  Meny thanks for the reply! I also got some of thees 
https://www.plusled.dk/zigbee-pcb-foh-7856/
But i cant get the to pair how did you manage that?

**Reply by chris-1243** [reply]
I did use the "permit join via a specific device" instead of All. You need a router able to manage ZigbeeGreen message not so far depending of your installation/network.

It ensure a smooth use of your ZG device and a strong connection as well.

I do use 7 PTM215Z in my network and some are quite far away from the ccordinator. Pairing them as describe above solved all my connectivity problem and now, they never fail (4 are connected via 2 Ikea Tradfri (V2) - [ICPSHC24-10EU-IL-1](https://www.zigbee2mqtt.io/devices/ICPSHC24-10EU-IL-1.html#ikea-icpshc24-10eu-il-1) and are the ones really far from my ccordinator).

**Reply by Daniel-PT** [reply]
@chris-1243  Thanks!
So you think it can use Sonoff 3.0 Zigbee controller ZBDongel plus with https://ledpaerer.dk/media/catalog/product/attachment/91-954.pdf ?

AM i doing something wroung when i cant see  "permit join via a specific device"
![image](https://github.com/Koenkk/zigbee2mqtt/assets/20338866/2b95307d-dfaa-4015-87b0-eb8df25e61b4)

Regards Daniel

**Reply by chris-1243** [reply]
Which model of Sonoff coordinator are you using? Model P (based on TI CC2652P) or Model E (based on EFR32MG21) ?

If P version: you need to add a router to your network (Hue light, sunricher modules, some Ikea lights). Those are able to "translate" ZG frames and send them to your coordinator.

E version should be the same even if I was able to pair a ZG device directly to the coordinator. I would anyway use a router to ensure a smooth connection.

On your router (device) those clusters (enfpoint 242) are important as based on my little experience
![image](https://github.com/Koenkk/zigbee2mqtt/assets/142902565/40c347dd-ee2e-4643-9b57-dda34d981b09)

**Reply by Daniel-PT** [reply]
I have the CC2652P model :) Ouh okay so i need somekind of bulp to be the translator? :(

**Reply by chris-1243** [reply]
Exactly. You need a translator.

Below the Zigbee2mqtt link for this specific device

[Paring PTM215Z](https://www.zigbee2mqtt.io/devices/PTM_215Z.html)

**Reply by Daniel-PT** [reply]
Meny thanks!

So is https://www.plusled.dk/zigbee-pcb-foh-7856/
The same as 
https://www.zigbee2mqtt.io/devices/PTM_215Z.html

I think it is a sunricher. The "battery plastic clip" says sunricher :)

**Reply by Daniel-PT** [reply]
Yea i can see that the name is 
Sunricher SR-ZGP2801K4-FOH-E

So i shold just pair a Sunricher SR-ZGP2801K4-FOH-E with my Sonoff and then it can controll my Ikea fyrtur blinds?
What about i dont have the "permit join via a specific device" ? :)

**Reply by chris-1243** [reply]
I am a bit confused regarding the model.

If it is a NON-battery powered, the link I provided should be correct.

If it is a battery powered device from Sunricher, you might consider this [link](https://www.zigbee2mqtt.io/devices/SR-ZGP2801K4-DIM.html).

As you will see, the pairing is not the same.

The "permit join via device" is exactly where you clicked on your screenshot. As soon as you will add a light (for example), then you will be able to pair a new device via this light by selecting it.
As I saw in your screenshot, you just started Z2M and you have not added any devices. This is why you have only "All" and "coordinator" in the list.

**Reply by Daniel-PT** [reply]
@chris-1243  meny thanks!

It is battery powered :) I will try to find a cheap ikea bulp :)

**Reply by chris-1243** [reply]
I would suggest this light from Ikea [LED2103G5](https://www.zigbee2mqtt.io/devices/LED2103G5.html#ikea-led2103g5). I run it in my network and some ZG devices were succesfully paired via this light

**Reply by Daniel-PT** [reply]
But when i think about it i dont really like the idea of that i need a "device" to be up and running so the outher devices work. If you can follow me ;)

**Reply by chris-1243** [reply]
Yep, I do understand.

I am not an expert at all. I guess it depends on how ZigbeeGreen is implemented in coordinator firmware, hardware or in Z2m as well. Based on that, it seems there is a need of a device (router which may be a lamp or plug) who is able to translate the ZG frames sent.
